modules/restoring-an-archived-pipeline-experiment.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="restoring-an-archived-pipeline-experiment_{context}"]
+= Restoring an archived pipeline experiment
+
+[role='_abstract']
+You can restore an archived pipeline experiment to the active state.
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group} ) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and has a pipeline server.
+* You have imported a pipeline to an active pipeline server.
```

Comment:
I think for the scope of archiving a pipeline experimented we can skip this. Experiments doesn't depends of pipelines

Reply:
Removed the unnecessary pre-req from line 19. See this commit https://github.com/opendatahub-io/opendatahub-documentation/pull/355/commits/4d73cb42c5115d1476d00680bea31f6b35827a8c.